packages/plugins/basic-pipeline/src/index.js

```diff
@@ -8,23 +8,24 @@ require('./env');
 
 class BasicPipeline {
 
-  constructor(embark, options) {
+  constructor(embark) {
     this.embark = embark;
     this.assetFiles = embark.config.assetFiles;
     this.isFirstBuild = true;
     this.embarkConfig = embark.config.embarkConfig;
     // TODO: why god why
-    this.useDashboard = options.useDashboard;
+    // this.useDashboard = options.useDashboard;
+    this.useDashboard = true;
```

Comment:
Why always `true`?

That would seem to have implications re: usage of `LongRunningProcessTimer` in this module relative to whether `--no-dashboard` was give in the cli.